cirq-web/cirq_ts/src/bloch_sphere/components/scene.ts

```diff
@@ -0,0 +1,122 @@
+import {Scene, PerspectiveCamera, WebGLRenderer, Camera, Object3D} from 'three';
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+
+export class BlochSphereScene {
+  private static readonly VIZ_WIDTH: number = 500;
+  private static readonly VIZ_HEIGHT: number = 500;
+
+  private scene: Scene;
+
+  /**
+   * The following declarations represent the Camera, Renderer,
+   * and Controls objects for the scene.
+   * camera - the Three.js camera object responsible for seeing the
+   *  visualization from different locations
+   * renderer - the Three.js renderer object resposible for rendering
+   *  the visualization into the browser. We will be using the WebGL
+   * renderer
+   * controls - the Three.js OrbitControls object reponsible for managing
+   *  mouse input and moving the camera, zooming in and out, etc.
+   * divId - the id of the HTML div that will contain the scene output
```

Comment:
This is not necessarily a div, it can be any container - `<div>` or `<span>` typically but maybe even `<body>`...I'd suggest renaming to `containerId`